extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -481,9 +486,17 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
-    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
-    const currentScrollTop = this.element./*OK*/scrollTop || 1;
-    this.element./*OK*/scrollTop = currentScrollTop;
+    const currentScrollTop = this.element./*OK*/scrollTop;
+
+    if (this.isIos_) {
+      // To avoid scroll-freeze issues in iOS, prevent reaching top/bottom
+      if (currentScrollTop == 0) {
+        this.element./*OK*/scrollTop = 1;
+      } else if (this.element./*OK*/scrollHeight ==
+            currentScrollTop + this.element./*OK*/offsetHeight) {
```

Comment:
As a temporary fix, this is probably fine. But it's likely to cause DOM recalcs if anything in the lightbox mutates. We might want to investigate caching the `scrollTop`/`offsetHeight` if we notice any issues.